cfme/containers/image_registry.py

```diff
@@ -40,3 +46,17 @@ def navigate(self, detail=True):
         else:
             sel.force_navigate('containers_image_registry',
                 context={'image_registry': self})
+
+    def get_detail(self, *ident):
```

Comment:
That doesn't make much sense to me.. In the test I need stuff like Container --> Properties --> Name value
How does provider.summary... help me get that value?
Any test that does something similar?